tests/foreman/cli/test_remoteexecution.py

```diff
@@ -177,9 +183,8 @@ def test_positive_run_custom_job_template_by_ip(self, rex_contenthost, module_or
 
         :parametrized: yes
         """
-        pass
```

Comment:
Does this exists in 6.10 and master, else we need to remove them there. Non blocking for this PR .

Reply:
ugh, great catch

Reply:
not in 6.10, but in master yes -- fixing here https://github.com/SatelliteQE/robottelo/pull/9484